src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -405,10 +405,11 @@ String getChangedFilesList(config) {
 
 def buildDocker(image, config) {
 
-    def img      = image.url
-    def arch     = image.arch
-    def filename = image.filename
-    def distro   = image.name
+    def img = image.url
+    def arch = image.arch
+    def filename = image.filename.toString()
```

Comment:
```image.filename.toString().trim()``` to cut trailing spaces
why toString needed at all?

Reply:
It is needed to pass condition `if (config.get("cFiles").contains(filename))`. The dockerfiles were changed, but re-built has not been happened. I will comment in the code that solution was found here: https://stackoverflow.com/q/56829842/3648361

Reply:
It works without `.trim()`, but I can add it.